.github/workflows/publish-artifacts.yml

```diff
@@ -86,7 +86,7 @@ jobs:
         cd oras
         cp ../target/${{ env.RUST_TARGET }}/release/attestation-agent .
         tar cJf attestation-agent.tar.xz attestation-agent
-        arch_tag="${{ github.sha }}-${{ matrix.platform.tee }}_${{ matrix.platform.arch }}"
+        arch_tag="${{ github.sha }}-${{ matrix.platform.tee }}_${OCI_ARCH}"
```

Comment:
@Xynnn007, Would this change have any implications for the images of the x86_64 and arm64 archs being consumed?